package.json

```
@@ -20,7 +20,7 @@
 		"react-dom": "18.2.0",
 		"react-helmet-async": "1.3.0",
 		"react-redux": "8.1.1",
-		"react-router-dom": "6.14.1"
+		"react-router-dom": "^6.14.1"
```

У вас скорее на защите бот будет ругаться если версии не зафиксированы 
```suggestion
		"react-router-dom": "6.14.1"
```